sections/gov/components/Panel.tsx

```diff
@@ -128,9 +127,9 @@ const Panel: React.FC<PanelProps> = () => {
 							setProposal(null);
 							setPanelType(PanelType.LIST);
 							if (activeTab === SPACE_KEY.PROPOSAL) {
-								router.push(ROUTES.Gov.Home);
+								router.back();
```

Comment:
Since both cases are calling back we wouldn't need the if statements. Does this work if I reload the page while on a proposal though? Or just visiting a proposal directly?

Reply:
Yup - updated.